include/drivers/console/uart_mux.h

```diff
@@ -128,6 +128,17 @@ typedef void (*uart_mux_cb_t)(const struct device *uart,
  */
 void uart_mux_foreach(uart_mux_cb_t cb, void *user_data);
 
+/**
+ * @brief Disable/enable the mux.
+ *
+ * @details Disable does not re-instate whatever ISRs and configs were present
+ * before the mux was enabled. This must be done by the user.
+ *
+ * @param dev UART mux device pointer
+ */
+void uart_mux_disable(const struct device *dev);
+void uart_mux_enable(const struct device *dev);
```

Comment:
Please provide documentation separately for both functions so that generated output looks ok

Reply:
It looks like the documentation change went to wrong commit, as I see them in the next commit. I know it is a pain but could you move them to here.

Reply:
ouff